string detector_ns
string robot_ns
geometry_msgs/PoseStamped pose_D_D
geometry_msgs/PoseStamped pose_R_D
float64 alpha
geometry_msgs/PoseStamped pose_D_R
geometry_msgs/PoseStamped pose_R_R
float64 beta
---
bool ack